### DISCUSSION TITLE
Sliding Window Inference

### DISCUSSION THREAD

**Discussion by aeye-slater** [Project-MONAI/tutorials#1637] [Q&A]
Our group is a little new to 3D segmentation and I am having trouble understanding why the sliding window inference is used in the tutorials.  I could not find references online to using this except within MONAI.

What I have gathered so far is you can train on smaller patches and then using the SWI to predict on larger images.  Is that the basic idea?

Thanks

**Comment by diazandr3s** [comment]
Hi @aeye-slater,

Thanks for posting this. It is a good question.

The sliding window technique has many advantages - among them is reducing complexity during training. One of the key features, as you said, is to train and run inference on patches instead of using whole volumes (in medical imaging applications). So there is no need to resize volumes when they are big.

Here is a blog post about other applications using sliding window: https://medium.com/@data-overload/sliding-window-technique-reduce-the-complexity-of-your-algorithm-5badb2cf432f

Hope this helps,
